data-prepper-plugins/otel-trace-source/src/main/java/com/amazon/dataprepper/plugins/source/oteltrace/OTelTraceSource.java

```diff
@@ -43,6 +44,7 @@
 @DataPrepperPlugin(name = "otel_trace_source", pluginType = Source.class, pluginConfigurationType = OTelTraceSourceConfig.class)
 public class OTelTraceSource implements Source<Record<Object>> {
     private static final Logger LOG = LoggerFactory.getLogger(OTelTraceSource.class);
+    private static final String HTTP_HEALTH_CHECK_PATH = "/health";
```

Comment:
This probably should be configurable. But, some health check is better than no health check in my opinion.

Reply:
yeah, we can make this configurable for customer to provide the path for health check.

Reply:
There might be value is making http health check path configurable. I don't see immediate use case for this. We can track this as separate issue to make path configurable.